### PR TITLE
chore: adding line breaks to PR template comments

### PR DESCRIPTION
**Pull request by arboleya**
# Summary

The title and nothing else.

# Checklist

- [ ] I **_added_** — `tests` to prove my changes
- [ ] I **_updated_** — all the necessary `docs`
- [x] I **_reviewed_** — the entire PR myself, using the GitHub UI
- [ ] I **_described_** — all breaking changes and the Migration Guide

### CODE REVIEW

File: apps/docs/src/guide/fuels-cli/commands.md [hunk 1]
```diff
@@ -180,3 +180,31 @@ npx fuels@{{fuels}} versions
 
 You have all the right versions! ⚡
 ```
+
+## `fuels info`
+
+Outputs useful information that can assist with possible nmisconfigurations and general information you can paste when creating new issues.
```
Comment (arboleya) [inline apps/docs/src/guide/fuels-cli/commands.md:186]:
```suggestion
Outputs helpful information that can assist with possible misconfigurations and general information you can paste when creating new issues.
```

**Review by nedsalk — APPROVED** [review]

**Review by maschad — DISMISSED** [review]

File: apps/docs/src/guide/fuels-cli/commands.md [hunk 2]
```diff
@@ -180,3 +180,31 @@ npx fuels@{{fuels}} versions
...
+    "systemForcVersion": "0.61.1",
+    "systemForcPath": "forc",
+    "systemFuelCoreVersion": "0.30.0",
+    "systemFuelCorePath": "fuel-core"
+  },
+  "nodeVersion": "v22.3.0",
+  "osType": "Darwin",
+  "osArch": "arm64"
+}
+```
```
Comment (petertonysmith94) [inline apps/docs/src/guide/fuels-cli/commands.md:210]: I assume this was mean't to go in:
- https://github.com/FuelLabs/fuels-ts/pull/2802

**Review by petertonysmith94 — COMMENTED** [review]
I

File: apps/docs/src/guide/fuels-cli/commands.md (hunk 2 above)
Comment (danielbate) [reply apps/docs/src/guide/fuels-cli/commands.md:210]: Accidental copy from [here](https://github.com/FuelLabs/fuels-ts/pull/2738#discussion_r1685566897) I believe.

**Review by nedsalk — APPROVED** [review]

**Review by petertonysmith94 — APPROVED** [review]

**Review by Torres-ssf — APPROVED** [review]

**Pull request merged**